### ISSUE TITLE
ambient access to setTimeout

### ISSUE THREAD

**Issue by dckc** [rchain-community/rgov#91]
looks like it was introduced in 509823f68725f6f

**Comment by dckc** [comment]
how is this a UI issue?

**Comment by Bill-Keuntje** [comment]
setTimeout is in participate.js -- which is the code for the developer UI.

**Comment by dckc** [comment]
But it's not visible to users. I would rather use the UI label to indicate issues that are visible to users.

And I'm not sure the developer UI even counts as a UI.

This is a code style issue and a (potential) security issue; you might label it as one or both of those.

**Comment by Bill-Keuntje** [comment]
Agreed, but since we have no "users" (yet), UI is for the UI team - not user feedback.

I think we need a "security" label as well...
